rexrov2_description/urdf/rexrov2_base.xacro

```diff
@@ -27,7 +27,7 @@
   <xacro:property name="mass" value="1862.87"/>
 
   <xacro:macro name="rexrov2_base"
-    params="namespace *gazebo use_simplified_mesh inertial_reference_frame">
+    params="namespace use_simplified_mesh inertial_reference_frame"> <!-- *gazebo -->
     <!-- Defining base link according to ROS conventions: x forward, z up -->
```

Comment:
These lines comments are creating conflict with `Command` substitution laucher

Reply:
I've tested and indeed, it is necessary not just to remove` <!-- *gazebo -->` as well as the following line `<!-- Defining base link according to ROS conventions: x forward, z up -->`.